style.css

```diff
@@ -155,9 +155,9 @@ nav {
   }
```

Comment:
- [ OPTIONAL] Well done implementing the suggested changes by the previous reviewer. To improve your user Interface I would suggest you remove the `padding-top` from the `.lastMsgText` selector on your desktop media query and also remove the `padding-left `from the `.lastMsgText`selector on the mobile media query.